hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/DeploymentManagement.java

```diff
@@ -48,71 +48,12 @@
+     * Assigns {@link DistributionSet}s to {@link Target}s according to the
+     * {@link DeploymentRequest}.
      *
-     * @param dsID
-     *            the ID of the distribution set to assign
-     * @param actionType
-     *            the type of the action to apply on the assignment
-     * @param forcedTimestamp
-     *            the time when the action should be forced, only necessary for
-     *            {@link ActionType#TIMEFORCED}
-     * @param controllerIDs
-     *            the IDs of the target to assign the distribution set
-     * @return the assignment result
-     *
-     * @throws IncompleteDistributionSetException
-     *             if mandatory {@link SoftwareModuleType} are not assigned as
-     *             define by the {@link DistributionSetType}.
-     *
-     * @throws EntityNotFoundException
-     *             if either provided {@link DistributionSet} or {@link Target}s
-     *             do not exist
-     * 
-     * @throws QuotaExceededException
-     *             if the maximum number of targets the distribution set can be
-     *             assigned to at once is exceeded
-     */
-    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_READ_REPOSITORY_AND_UPDATE_TARGET)
-    DistributionSetAssignmentResult assignDistributionSet(long dsID, @NotNull ActionType actionType,
-            long forcedTimestamp, @NotEmpty Collection<String> controllerIDs);
-
-    /**
-     * Assigns the addressed {@link DistributionSet} to all {@link Target}s by
-     * their IDs with a specific {@link ActionType} and {@code forcetime}.
-     *
-     * @param dsID
-     *            the ID of the distribution set to assign
-     * @param targets
-     *            a list of all targets and their action type
-     * @return the assignment result
-     *
-     * @throws IncompleteDistributionSetException
-     *             if mandatory {@link SoftwareModuleType} are not assigned as
-     *             define by the {@link DistributionSetType}.
-     *
-     * @throws EntityNotFoundException
-     *             if either provided {@link DistributionSet} or {@link Target}s
-     *             do not exist
-     * 
-     * @throws QuotaExceededException
-     *             if the maximum number of targets the distribution set can be
-     *             assigned to at once is exceeded
-     */
-    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_READ_REPOSITORY_AND_UPDATE_TARGET)
-    DistributionSetAssignmentResult assignDistributionSet(long dsID,
-            @NotEmpty Collection<TargetWithActionType> targets);
-
-    /**
-     * Assigns the given set of {@link DistributionSet} entities to the given
-     * {@link Target}s using the specified {@link ActionType} and
-     * {@code forcetime}.
-     *
-     * @param dsIDs
+     * @param deploymentRequests
      *            the set of IDs of the distribution sets to assign
-     * @param targets
-     *            a list of all targets and their action type
+     * 
```

Comment:
javadoc misleading... should be sth like "the set of deployment requests specifying the distribtution set assignments..."